Wrappers/Python/cil/optimisation/functions/IndicatorBox.py

```diff
@@ -29,67 +35,389 @@ class IndicatorBox(Function):
                                             0, \text{ if } x \in [a, b] \\
                                             \infty, \text{otherwise}
                                      \end{cases}
-    
+        
+        Parameters
+        ----------
+        
+            lower : float, DataContainer or numpy array, default None
+                Lower bound. If set to None, it is equivalent to ``-np.inf``.
+            upper : float, DataContainer or numpy array, default None
+                Upper bound. If set to None, it is equivalent to ``np.inf``.
+            backend : string, default 'numba'
```

Comment:
for slicer I went with `accelerated=True` rather than `backend='C'`. This might be a more general convention, but doesn't exactly tell the user much...